axon-server/administration/admin-configuration/configuration.md

```diff
@@ -0,0 +1,1072 @@
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.secondary-cleanup-delay</td>
+      <td style="text-align:left">Windows only. Delay before forcing the other segment files from memory
+        when no longer in use.</td>
+      <td style="text-align:left">30</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.segment-size</td>
+      <td style="text-align:left">Size of a transaction log file.</td>
+      <td style="text-align:left">16MB</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.snapshot-flow-buffer</td>
+      <td style="text-align:left">Number of unconfirmed install snapshot messages that may be sent to peer.</td>
+      <td
+      style="text-align:left">50</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.sync-interval</td>
+      <td style="text-align:left">Interval to check for files that can be closed.</td>
+      <td style="text-align:left">1000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.wait-for-leader-timeout</td>
+      <td style="text-align:left">Timeout to wait for leader when requesting access to event store while
+        leader change in progress, if not set defaults to maxElectionTimeout.</td>
+      <td
+      style="text-align:left">-1</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"><em><b>Keep Alive</b></em>
+      </td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">keep-alive-time</td>
+      <td style="text-align:left">Interval at which AxonServer will send timeout messages. Set to 0 to disbable
+        gRPC timeout checks.</td>
+      <td style="text-align:left">2500</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">keep-alive-timeout</td>
+      <td style="text-align:left">Timeout for keep alive messages on gRPC connections.</td>
+      <td style="text-align:left">5000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">min-keep-alive-time</td>
+      <td style="text-align:left">Minimum keep alive interval accepted by this end of the gRPC connection.</td>
+      <td
+      style="text-align:left">1000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">client-heartbeat-timeout</td>
+      <td style="text-align:left">Timeout on application level heartbeat between client and Axon Server.</td>
+      <td
+      style="text-align:left">5000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">client-heartbeat-check-initial-delay</td>
```

Comment:
Missing description for this and the upcoming two points.
Added, this listing would also benefit from a time unit.